docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -51,7 +51,7 @@ To get started:
 * annotate your entities with `@Entity` and make them extend `PanacheEntity`
 * place your entity logic in static methods in your entities
 
-Follow the link:hibernate-orm.html#setting-up-and-configuring-hibernate-orm-without-persistence-xml-recommended[Hibernate set-up guide for all configuration].
+Follow the link:hibernate-orm#setting-up-and-configuring-hibernate-orm-without-persistence-xml-recommended[Hibernate set-up guide for all configuration].
```

Comment:
I think this specific anchor is broken now - it exists currently but I need to check after my own ORM guides PR if this one still exists with same name.

Not your PR's fault, so let's merge it - I'll send a follow up.

Reply:
https://github.com/quarkusio/quarkus/pull/5263